### PR TITLE
Demo/login with email remove session

### PR DESCRIPTION
**Pull request by HilaPartukTS**

### CODE REVIEW

File: login-with-email/backend/index.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import express from 'express'
 import fetch from 'node-fetch'
 const router = express.Router()
 
+let accessToken = null;
```
Comment (ts-ron) [inline login-with-email/backend/index.js]: Add comment: "In a production server, you would cache the access token, and regenerate whenever it expires. The below emulates this 'cache'"

File: sample.env [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # duplicate and rename this file to .env
-# Copy the base codespave URL, and add -8080.preview.app before the github.dev suffix. This is the base launch URL for configurations
```
Comment (ts-ron) [inline sample.env:2]: 🤦 , 🙏

**Review by ts-ron — APPROVED** [review]
LGTM

File: login-with-email/backend/index.js (hunk 1 above)
Comment (HilaPartukTS) [reply login-with-email/backend/index.js:8]: added

**Pull request merged**